cognite/client/_api/workflows.py

```diff
@@ -56,29 +56,29 @@ def wrap_workflow_ids(
 class WorkflowTriggerAPI(APIClient):
     _RESOURCE_PATH = "/workflows/triggers"
 
-    def create(
+    def upsert(
```

Comment:
This looks breaking?

Reply:
It is. What's the process with the SDK?

Reply:
Discussed on slack, will keep around with deprecation warning until next major version